client_plugin/utils/config/config.go

```diff
@@ -51,6 +51,7 @@ const (
 // Config stores the configuration for the plugin
 type Config struct {
 	Driver        string `json:",omitempty"`
+	SlaveDriver   string `json:",omitempty"`
```

Comment:
This should not be here.
This is a shared plugin's specific flag.
You should put it inside struct `VolumeDriver` shared_driver.go and read it through flag.
Check how `project` is used inside photon_driver.go.